src/promptflow/promptflow/executor/_line_execution_process_pool.py

```diff
@@ -346,13 +348,20 @@ def run(self, batch_inputs):
                 self._nlines,
             ),
         ):
-            self._pool.starmap(
-                self._timeout_process_wrapper,
-                [
-                    (run_start_time, self._inputs_queue, self._line_timeout_sec, result_list)
-                    for _ in range(self._n_process)
-                ],
-            )
+            try:
+                self._pool.starmap(
+                    self._timeout_process_wrapper,
+                    [
+                        (run_start_time, self._inputs_queue, self._line_timeout_sec, result_list)
+                        for _ in range(self._n_process)
+                    ],
+                )
+            except Exception as e:
+                logger.error(f"Process {os.getpid()} failed with exception: {e}")
+                raise ProcessPoolError(
+                    message_format=f"Process {os.getpid()} failed with exception: {e}",
+                    target=ErrorTarget.EXECUTOR,
+                )
```

Comment:
Add `from e` so the traceback can include the original error.

Reply:
fixed